.github/workflows/dotnet.yml

```diff
@@ -1,6 +1,9 @@
 name: .NET
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [ main ]
```

Comment:
This will not run the CI if I push a branch without having a PR open yet though, no? 🥲

Reply:
> This will not run the CI if I push a branch without having a PR open yet though

Yes. I assume if you pushed a branch you'll eventually open a PR?

Reply:
Eventually, but I might very well also have branches with no PR open if they're still tests/work in progress

Reply:
Then no way to avoid duplication. I'm going to close

Reply:
That's a bummer, I really hope we'll find out a way to properly fix this at some point 🥲
Thank you for trying though!